OpenRA.Mods.Common/Traits/World/EditorActorPreview.cs

```diff
@@ -76,6 +79,8 @@ public EditorActorPreview(WorldRenderer worldRenderer, string id, ActorReference
 			Tooltip = (tooltip == null ? " < " + Info.Name + " >" : tooltip.Name) + "\n" + owner.Name + " (" + owner.Faction + ")"
 				+ "\nID: " + ID + "\nType: " + Info.Name;
 
+			TypeSummary = Info.Name + (tooltip != null ? " (" + tooltip.Name + ")" : string.Empty);
```

Comment:
This adds a double `((` for things like the camera where there is already a `(` in the name.